tests/test_fetch_events.py

```diff
@@ -0,0 +1,130 @@
+from collections.abc import Generator
+from contextlib import contextmanager
+from datetime import datetime, timedelta
+from uuid import uuid4
+
+import pytest
+from ata_db_models.models import Event, SQLModel
+from sqlalchemy.engine import Engine
+from sqlalchemy.orm import sessionmaker
+
+from ata_pipeline1.fetch_events import fetch_events
+from ata_pipeline1.helpers.enums import EventName, SiteName
+
+
+@pytest.fixture
+def single_event() -> Event:
+    return Event(
+        site_name=SiteName.AFRO_LA,
+        derived_tstamp=datetime.now(),
```

Comment:
💭 Maybe create a fixture for timestamp and just pass it to whatever uses `datetime.now()`?